docs/quickstart/generic.md

```diff
@@ -59,3 +64,5 @@ cd gcp-linux # modify this line as needed to access the template
 vim main.tf
 coder templates update gcp-linux # updates the template
 ```
+
+> See our [template documentation](../templates.md) for more information.
```

Comment:
Use the Up next pattern for pretty rendering...

```
## Up next

- Learn about [templates](../templates.md)
```

<img width="482" alt="Screen Shot 2022-08-09 at 6 54 13 PM" src="https://user-images.githubusercontent.com/7416144/183781393-c2f771e3-d904-40c7-9972-d6ac535dce79.png">